src/ape_networks/_cli.py

```diff
@@ -14,7 +14,7 @@ def _filter_option(name: str, options):
     return click.option(
         f"--{name}",
         f"{name}_filter",
-        multiple=Tree,
```

Comment:
lol, how did this happen??

Reply:
It wasn't causing any issues, but I am guessing it happens from a text editor autocorrection overlook. Like went to type `True` but the IDE autofilled to `Tree` since it was present in the namespace. That is my guess!